rnd/autogpt_builder/src/components/Flow.tsx

```diff
@@ -253,6 +255,7 @@ const FlowEditor: React.FC<{ flowID?: string; className?: string }> = ({
     })));
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
?

Reply:
just put this in for now til i do a proper linting and formatting PR to get the build working